relayer-engine/packages/relayer-plugin-interface/src/index.ts

```diff
@@ -30,6 +30,15 @@ export interface ChainConfigInfo {
   chainId: ChainId;
   chainName: string;
   nodeUrl: string;
+  bridgeAddress?: string;
+  wrappedAsset?: string | null;
+  tokenBridgeAddress?: string;
+  nativeCurrencySymbol?: string;  
+  terraName?: string; 
+  terraChainId?: string;
+  terraCoin?: string;
+  terraGasPriceUrl?: string;
+  isTerraClassic?: boolean;
```

Comment:
Why does this config exist for every chain? can this be defined for any evm chain, solana, algorand etc?

Reply:
For all the ones that have a `terra` prefix the answer is not... It *needs* to be defined for a terra chain, but it shouldn't be present for configs of another chains
For the ones not prefixed, it should existe for all of them. Buuuuut, this one of the things we though we should improve with giorgio. As Andreas mentioned in [this comment](https://github.com/wormhole-foundation/relayer-engine/pull/82/files#r1141408350) we don't need to get this information from the client since we already have it at our disposal in the wormhole-sdk, so we should probably get it from there.
We were thinking about this with Gio, but we didn't want to get out of scope... If you think it's better to clean this up now (i'd agree) I'm happy to refactor this! :muscle: